nuxeo/nuxeo.py

```diff
@@ -713,6 +713,23 @@ def request(
 
         return self._read_response(resp, url)
 
+    def drive_config(self):
+        """
+        Fetch the Drive configuration file at $NUXEO_URL/api/v1/drive/configuration.
```

Comment:
"file" can be removed, technically it's not a file but just some JSON content in the response body

Reply:
:+1: 
An update is ongoing :)